### PR TITLE
Improved server command and added 'server_settings'

### PR DESCRIPTION
**Pull request by masipcat**
I'm splitting the changes in https://github.com/plone/guillotina/pull/764

### CODE REVIEW

File: guillotina/commands/server.py [hunk 1]
```diff
@@ -21,33 +21,37 @@ def get_parser(self):
         parser.add_argument("--asgi-server", default="uvicorn", type=str)
         return parser
 
-    def run(self, arguments, settings, app):
-        port = arguments.port or settings.get("address", settings.get("port"))
-        host = arguments.host or settings.get("host", "0.0.0.0")
+    async def run(self, arguments, settings, app):
+        host = arguments.host or app.settings.get("host", "0.0.0.0")
```
Comment (bloodbare) [inline guillotina/commands/server.py]: This breaks all actual configs!

Comment (masipcat) [reply guillotina/commands/server.py]: which thing? I just swiped the lines

Comment (bloodbare) [reply guillotina/commands/server.py]: app.settings is mapped to server_settings, isn't it?

Comment (masipcat) [reply guillotina/commands/server.py]: not, it's mapped to app_settings (after being merge with all the configs provided by the user) https://github.com/plone/guillotina/pull/781/files#diff-9aee2a99444a30b44f1a3a9ce2b1d76aR264

Comment (bloodbare) [reply guillotina/commands/server.py]: Sorry confused with typing

**Pull request merged**